tests/js/client/load-balancing/load-balancing-pregel-auth-cluster.js

```diff
@@ -138,39 +137,50 @@ function PregelAuthSuite () {
-      assertTrue(result.body.state === "loading" || result.body.state === "running");
-      // check if we've updated the persisted document state properly
-      let persistedState = db[pregelSystemCollection].document(taskId);
-      assertTrue(persistedState.data.state === "loading" || persistedState.data.state === "running");
-
-      require('internal').wait(5.0, false);
-
-      url = `${baseUrl}/${taskId}`;
-      result = sendRequest(users[0], 'DELETE', url, {}, {}, false);
-
-      assertFalse(result === undefined || result === {});
-      assertFalse(result.body.error);
-      assertEqual(result.status, 200);
-
-      // check if we've updated the persisted document state properly after deletion
-      persistedState = db[pregelSystemCollection].document(taskId);
-      assertTrue(persistedState.data.state === "canceled");
+      const postResponse = sendRequest(users[0], 'POST', baseUrl, task, true);
+      assertFalse(postResponse === undefined || postResponse === {});
+      assertEqual(postResponse.status, 200);
+      const pid = postResponse.body;
+      assertTrue(pid > 1);
+
+      // check that pregel status of run can be retrieved
+      // need to wait for short time until pregel run was created
+      const sleepIntervalSeconds = 0.2;
+      const maxWaitSeconds = 10;
+      let wakeupsLeft = maxWaitSeconds / sleepIntervalSeconds;
+      var statusResponse;
+      do {
+        require("internal").sleep(sleepIntervalSeconds);
+        statusResponse = sendRequest(users[0], 'GET', `${baseUrl}/${pid}`, {}, false);
+        assertFalse(statusResponse === undefined || statusResponse === {});
+        if (wakeupsLeft-- === 0) {
+          assertTrue(false, "Cannot retrieve status of pregel run, got response code " + statusResponse.status);
+          return;
+        }
+      } while (statusResponse.status !== 200);
+
+      const deleteResponse = sendRequest(users[0], 'DELETE',  `${baseUrl}/${pid}`, {}, {}, false);
+      assertFalse(deleteResponse === undefined || deleteResponse === {});
+      assertFalse(deleteResponse.body.error);
+      assertEqual(deleteResponse.status, 200);
+
+      // check that pregel status state is canceled
+      // need to wait for short time until pregel run was canceled
+      wakeupsLeft = maxWaitSeconds / sleepIntervalSeconds;
+      do {
+        require("internal").sleep(sleepIntervalSeconds);
+        statusResponse = sendRequest(users[0], 'GET', `${baseUrl}/${pid}`, {}, false);
+        assertFalse(statusResponse === undefined || statusResponse === {});
+        if (wakeupsLeft-- === 0) {
+          assertTrue(false, "Pregel run was not canceled successfully but is in state " + statusResponse.body.state);
+          return;
+        }
+      } while (!pregelTestHelpers.runCanceled(statusResponse.body));
     },
 
     testPregelForwardingDifferentUser: function() {
-      let url = baseUrl;
+      const runCreator = users[0];
+      const differentUser = users[1];
```

Comment:
```suggestion
      const runUser = users[0];
      const differentUser = users[1];
```

Very very minor suggestion :P